### PR TITLE
AWS Native Utilization with params and Tag Support

### PR DESCRIPTION
**Pull request by rshade**
### Description

This pull request cleans the auth requests and adds in tag support. 

### Issues Resolved

[List any existing issues this PR resolves]

### Contribution Check List

- [ ] All tests pass.
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable
- [ ] New functionality has been documented in CHANGELOG.MD

### CODE REVIEW

**Comment by rshade** [comment]
https://cl.ly/84f0819c4412

File: cost/aws/instance_cloudwatch_utilization/README.md [hunk 1]
```diff
@@ -11,13 +11,20 @@ This Policy Template gathers AWS CloudWatch data for instances on 30 day interva
 
 ### Functional Details
 
-- This policy uses RightScale to get a list of instances, it then polls aws CloudWatch for metrics for instance performance and delivers a report. If you get an **N/A** in a field you will need to install the [CloudWatch Agent](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/Install-CloudWatch-Agent.html) on the instance to get those metrics. 
+- This policy identifies all instances reporting performance metrics to Log Analytics whose CPU or Memory utilization is below the thresholds set in the **Average used memory percentage** and **Average used CPU percentage** parameters.
```
Comment (srpomeroy) [inline cost/aws/instance_cloudwatch_utilization/README.md]: Cloud Watch not Log Analytics

File: cost/aws/instance_cloudwatch_utilization/aws_instance_cloudwatch_utilization.pt [hunk 2]
```diff
@@ -663,4 +363,62 @@ end
...
+escalation "tag_resources" do
+  run "tag_resources", data, $param_take_action_tag
+end
+
+###############################################################################
+# Cloud Workflow
+###############################################################################
+
+define tag_resources($data, $param_take_action_tag) return $all_responses do
+  $$debug=true
```
Comment (srpomeroy) [inline cost/aws/instance_cloudwatch_utilization/aws_instance_cloudwatch_utilization.pt]: Do we want to leave this as `true`?

File: cost/aws/instance_cloudwatch_utilization/aws_instance_cloudwatch_utilization.pt [hunk 3]
```diff
@@ -643,16 +341,18 @@ policy "pol_utilization" do
     summary_template "{{ rs_project_name }} (Account ID: {{ rs_project_id }}): {{ len data }} rows containing AWS instance CloudWatch utilization data"
     detail_template <<-EOS
 # AWS Instance Utilization
-| Region | instanceID | Hostname | Private DNS Name | CPU Maximum % | CPU Minimum % | CPU Average % | Disk Maximum % | Disk Minimum % | Disk Average % | Memory Minimum % | Memory Maximum % | Memory Average % |
-| ------ | ---------- | -------- | ---------------- | ------------- | ------------- | ------------- | -------------- | -------------- | -------------- | ---------------- | ---------------- | ---------------- |
+| Region | instanceID | Hostname | Private DNS Name | CPU Maximum % | CPU Minimum % | CPU Average % | Memory Minimum % | Memory Maximum % | Memory Average % |
```
Comment (srpomeroy) [inline cost/aws/instance_cloudwatch_utilization/aws_instance_cloudwatch_utilization.pt]: We are only displaying CPU Average, CPU Max, Memory Average, and Memory Max in the Log Analytics policy. We should consider removing the minimum metric.

**Pull request merged**